app/controllers/hostgroups_controller.rb

```diff
@@ -77,9 +76,10 @@ def destroy
 
   def current_parameters
     Taxonomy.as_taxonomy @organization, @location do
-      inherited_parameters = Hostgroup.find(params['hostgroup_parent_id']).inherited_params(true) if params['hostgroup_parent_id']
+      hostgroup_parent_parameters = Hostgroup.authorized(:view_hostgroups).find(params['hostgroup_parent_id']).parameters(true) if params['hostgroup_parent_id']
+      hostgroup_parameters = params['hostgroup_id'] ? Hostgroup.authorized(:view_hostgroups).find(params['hostgroup_id']).group_parameters : []
```

Comment:
and on this line (fixing the above error), `params['hostgroup_id']` is actually `'null'` for a new, unsaved host group